packages/tenderly-utils/e2e/tests.spec.ts

```diff
@@ -1,55 +1,44 @@
-import { input, rawlist } from '@inquirer/prompts'
 import * as Tenderly from '../src/tenderly'
 import * as Utils from '../src/utils'
 
-type Tokens = 'CBETH' | 'DAI' | 'ETH' | 'USDC' | 'RETH' | 'SDAI' | 'WBTC' | 'WSTETH'
+describe('e2e', () => {
+  it('should test fork creation and cleanup', async () => {
+    const atBlock = 19475802
+    const network = 'mainnet'
+    const walletAddress = '0x275f568287595D30E216b618da37897f4bbaB1B6'
 
-let walletAddress: string
-;(async () => {
-  const network: 'mainnet' | 'optimism' | 'arbitrum' | 'base' = await rawlist({
-    message: 'Select a network',
-    choices: [
-      { name: 'Mainnet', value: 'mainnet' },
-      { name: 'Arbitrum', value: 'arbitrum' },
-      { name: 'Base', value: 'base' },
-      { name: 'Optimism', value: 'optimism' },
-    ],
-  })
-
-  const tokens = Object.keys(Utils.tokenAddresses[network]) as Tokens[]
-
-  const resp = await Tenderly.createFork({ network })
-  const forkId = resp.data.root_transaction.fork_id
+    const tokens = Object.keys(Utils.tokenAddresses[network]) as Utils.TokenName[]
+    const resp = await Tenderly.createFork({ network, atBlock })
+    const forkId = resp.data.root_transaction.fork_id
 
-  walletAddress = await input({ message: 'Enter your wallet address: ' })
-
-  await Tenderly.setETHBalance({
-    forkId,
-    balance: Utils.tokenBalances['ETH'],
-    walletAddress: walletAddress,
-  })
-
-  for (const token of tokens) {
-    await Tenderly.setERC20TokenBalance({
+    await Tenderly.setETHBalance({
       forkId,
-      network,
-      token,
-      balance: Utils.tokenBalances[token],
+      balance: Utils.tokenBalances['ETH'],
       walletAddress: walletAddress,
     })
-  }
 
-  console.log(`Selected network: ${network}`)
-  console.log(`Wallet address: ${walletAddress}`)
-  console.log('-------------------------------------')
-  console.log('*** FUNDS ***')
-  console.log(`ETH - ${Utils.tokenBalances['ETH']}`)
-  tokens.forEach((token) => {
-    console.log(`${token} - ${Utils.tokenBalances[token]}`)
-  })
-  console.log('-------------------------------------')
-  console.log('Fork created: ', `https://rpc.tenderly.co/fork/${forkId}`)
+    for (const token of tokens) {
+      await Tenderly.setERC20TokenBalance({
+        forkId,
+        network,
+        token,
+        balance: Utils.tokenBalances[token],
+        walletAddress: walletAddress,
+      })
+    }
+
+    console.log(`Selected network: ${network}`)
```

Comment:
I think that we should expect the values instead of logging them, as otherwise the test is not properly validated

Reply:
This is not a test per se, it's just a script template so we can inspect the values .